api/nodes/views.py

```diff
@@ -1367,6 +1371,168 @@ def get_object(self):
         return fobj
 
 
+class NodeAddonList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin, NodeMixin):
+    """List of addons connected to this node *Read-only*
+
+    TODO: More Docs
+    """
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        ContributorOrPublic,
+        ExcludeRetractions,
```

Comment:
This will probably change to ExcludeWithdrawals or similar soon. Keep an eye out when you merge with develop.